Mathlib/GroupTheory/Sylow.lean

```diff
@@ -309,36 +303,32 @@ instance [hp : Fact p.Prime] [Finite (Sylow p G)] : IsPretransitive G (Sylow p G
       apply Q.2.nonempty_fixed_point_of_prime_not_dvd_card
       refine fun h => hp.out.not_dvd_one (Nat.modEq_zero_iff_dvd.mp ?_)
       calc
-        1 = card (fixedPoints P (orbit G P)) := ?_
-        _ ≡ card (orbit G P) [MOD p] := (P.2.card_modEq_card_fixedPoints (orbit G P)).symm
+        1 = Nat.card (fixedPoints P (orbit G P)) := ?_
+        _ ≡ Nat.card (orbit G P) [MOD p] := (P.2.card_modEq_card_fixedPoints (orbit G P)).symm
         _ ≡ 0 [MOD p] := Nat.modEq_zero_iff_dvd.mpr h
-      rw [← Set.card_singleton (⟨P, mem_orbit_self P⟩ : orbit G P)]
-      refine card_congr' (congr_arg _ (Eq.symm ?_))
+      rw [← Nat.card_unique (α := ({⟨P, mem_orbit_self P⟩} : Set (orbit G P))), eq_comm]
+      congr
       rw [Set.eq_singleton_iff_unique_mem]
       exact ⟨H.mpr rfl, fun R h => Subtype.ext (Sylow.ext (H.mp h))⟩⟩
 
 variable (p) (G)
 
 /-- A generalization of **Sylow's third theorem**.
   If the number of Sylow `p`-subgroups is finite, then it is congruent to `1` modulo `p`. -/
-theorem card_sylow_modEq_one [Fact p.Prime] [Fintype (Sylow p G)] :
-    card (Sylow p G) ≡ 1 [MOD p] := by
+theorem card_sylow_modEq_one [Fact p.Prime] [Finite (Sylow p G)] :
+    Nat.card (Sylow p G) ≡ 1 [MOD p] := by
```

Comment:
Is the file already in a shape where `open Fintype` can be removed?

Reply:
Yes (whereas PGroup still has an annoying proof that benefits from `open Fintype`).